### DISCUSSION TITLE
Which Quickstarts do you need?

### DISCUSSION THREAD

**Discussion by mffap** [zitadel/zitadel#1717] [Q&A]
Do you miss Quickstarts for any languages, tools, frameworks? Let us know below.

You can find all [Quickstarts here](https://docs.zitadel.ch/docs/quickstarts/introduction).

**Comment by fforootd** [comment]
I think `spring security` would be interesting.
With version 5.5.0 and newer they support `JWT Profile`

**Comment by gc-ss** [comment]
> Do you miss Quickstarts for any languages, tools, frameworks? Let us know below.

Yes!

Python3.8

https://github.com/caos/zitadel/issues/2062

If I were to start integrating with Zitadel in my Python3.8 project, what are some first steps I should take?

**Reply by fforootd** [reply]
This might be for @mffap

**Reply by mffap** [reply]
@gc-ss I can provide you with a sample script in Python 3.8 that shows you how to authenticate a service user (i.e. python script) with ZITADEL and make authenticated calls to the Management API to interact with your Org.
Would that be helpful?

In case you need authentication of users via user interface (e.g. django), we can figure out what the best oidc lib would be and get an example working as well, I guess.

**Reply by gc-ss** [reply]
> authenticate a service user (i.e. python script)
> Would that be helpful?

Yes, very much!

For service authentication my immediate usecases are:

i. I upload a script (that uses an API I built) to GitHub/GitHub Gist and anyone interested can download it. To run it they need an API key from me
ii. ZITADEL helps me manage that API key

> In case you need authentication of users via user interface (e.g. django), we can figure out what the best oidc lib would be and get an example working as well, I guess

Well, for user side authentication my immediate usecases are:

1. authentication via OAuth like GitHub, Google
2. Magic links via email

**Reply by mffap** [reply]
@gc-ss I've added an example here: 
https://github.com/caos/zitadel-examples/tree/main/python3/service_user

Have a look and let me know if it works. And of course, if you have any suggestions on how to improve it, then all contributions are welcome. Would be great to enhance the python examples.

**Reply by gc-ss** [reply]
Appreciate it! I'l get on it as soon as I'm unblocked from https://github.com/caos/zitadel/issues/2063

**Reply by mffap** [reply]
Actually we deprecated the old python3 example. That was only for accessing our own APIs, which in most cases is not as helpful. 

We just published an API example with Python3 + Flask. Hope this helps. We're also looking into publishing a Web-App version, but also open for contributions :)

https://github.com/zitadel/example-api-python3-flask

**Comment by fforootd** [comment]
I guess something like https://github.com/thomseddon/traefik-forward-auth

Would be interesting as well. Input came from here https://gitter.im/hashicorp-nomad/Lobby

**Comment by mirzaprangon** [comment]
Hello

Is there any quickstart for spring boot 3 and the new spring security 6? As you know, spring is used in many enterprises and is a big thing. And going with the latest java LTS version 17 will be a plus.

Would love to see some quickstarts for spring boot 3, security 6 and using multitenant. Multitentancy is a thing zitadel says it does way better that the realm approach keycloak takes. Will be awesome to see that in some quickstart.

**Reply by hifabienne** [reply]
Hei @mirzaprangon 
Thanks for your feedback.
We currently have only one Java (version 2.6.7) example: https://github.com/zitadel/zitadel-java
We just started the discussion about more examples, what to include in the first part and also to review the ones we have. When reviewing the examples we will have a look to into updating the versions as well.
https://github.com/zitadel/zitadel/issues/5124

**Reply by mirzaprangon** [reply]
Thank you for the information. I already have seen the java example, which is in boot 2.6 and java11.

Also, FYI, as far as I can understand(am no expert on spring), a lot has changed in the spring security 6.
